extensions/measurement-tracking/src/viewports/TrackedCornerstoneViewport.js

```diff
@@ -278,6 +283,13 @@ function TrackedCornerstoneViewport({
           viewportIndex={viewportIndex}
           imageIds={imageIds}
           imageIdIndex={currentImageIdIndex}
+          onNewImage={({ currentImageIdIndex }) => {
+            viewportGridService.setDisplaysetForViewport({
+              viewportIndex: activeViewportIndex,
+              displaySetInstanceUID: displaySet.displaySetInstanceUID,
+              frameIndex: currentImageIdIndex
+            });
+          }}
```

Comment:
Our app isn't super optimized yet. This triggers a rapid cascade of re-renders.
We could keep with a liberal throttle/debounce.

Reply:
@dannyrb i forgot to use the debounce, its already available in rcv. Updating...